index.rst

```diff
@@ -4,22 +4,215 @@
+It is anticipated that very few SIT-Com generated artifacts will fall into this category.
+
+SIT-Com Level Change Control
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the case of documentation that is applicable to only SIT-Com personnel and is internal to the SIT-Com subsystem(s), the implementation of a SIT-Com change board has been discussed but is not currently in place.
+Examples of documents that are SIT-Com specific but should be under change control include the documentation of workflows (like this one), management plans, and/or SIT-Com specific procedures that require sign-off by specific domain experts.
+Until such a system is put in place, these types of documentation exist as `Technotes`_.
+
+When required to be under stricter change controls, branch protection should be put in place.
+The designation of specific reviewers can also be manually specified.
+In the event that this requires higher-level SIT-Com repo permissions, please consult the ``#rubinobs-sitcom-docs`` Slack channel for support.
+
+General Documentation Types
+===========================
+
+This section outlines the different types of documentation used by SIT-Com personel.
+
+Jira Tickets
+^^^^^^^^^^^^
+
+SIT-Com related work should follow the `SIT-Com specific Jira workflow <https://sitcomtn-023.lsst.io/>`_.
+Documentation in Jira tickets should be sufficiently detailed such that it addresses the description in the task.
+It is not expected to be consulted in the future, especially on a regular occurrence. 
+In many cases, specifically when characterization activities have taken place over a series of tickets, external documentation should be developed and linked to the ticket(s).
+The type of document to be created is case dependent, but will most likely be covered by `Technotes`_ and/or inclusion of a section in the `Obs-Ops Webspace`_.
+The creation of the documentation can be performed under one of the already opened tickets, or, a new ticket can be created that explicitly deals with the generation of documentation.
+
+
+Verification Documentation and Artifacts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Systems Engineering team uses a specific Jira project for its verification performance and tracking.
+Personnel looking for verification documentation should consult `LSE-160 <ls.st/LSE-160>`_.
+
+The verification and validation process also utilize a specific repository for the storing of their notebooks that act as official artifacts.
+This is further described in the `Verification And Validation Notebooks`_ subsection below.
+
+
+Technotes
+^^^^^^^^^
+
+Technotes are useful for documenting outcomes of specific tests and/or findings, proposing new functionality or architectures, and/or documenting specific usages that augment user documentation.
+They can also be used as logs and/or status of longer term analyses.
+An example of this is the technote on Building and and fitting the pointing model for the AuxTel (`TSTN-014 <https://tstn-014.lsst.io/>`_.)
```

Comment:
"Building and and fitting" -> "Building and fitting"

I think there's a nicer way to reference tech notes. Try using

```
:tstn:`014`
```